src/components/project-details/readme-card.tsx

```diff
@@ -42,5 +42,5 @@ const ReadmeContent = ({ project }: Props) => {
 
   if (!html) return <Spinner />;
 
-  return <div dangerouslySetInnerHTML={{ __html: html }} />;
+  return <div dangerouslySetInnerHTML={{ __html: html as string }} />;
```

Comment:
Thank you @nvh95 actually the`html` you can see here comes from a serverless function I developed so I suppose it would be better to do that step at the server level, rather than in the browser?

Reply:
@michaelrambeau I know but if your server/ serverless/ db get compromised, bad people can alter your data (e.g: inject script to html) and our users can be at risk.
We should as many defense layers as possibles. This technique is called "Defense in depth", if one layer down, we have other layers to protect our users.

Reply:
Thank you, that's important indeed, I will do that in a separate PR, since it's not related to this one.

The line shows up in the Git diff because of a TypeScript error I got when I edited the file to throw a random error somewhere, which makes me think that it seems the compiler is stricter when you edit a file, I have to investigate more...

Reply:
I see. We can leave it in the future PR.